frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/premium.embedding.unit.spec.tsx

```diff
@@ -234,6 +236,9 @@ describe("[EE, with token] embedding settings", () => {
         expect(
           screen.getByLabelText("Enable Interactive embedding"),
         ).not.toBeChecked();
+        await userEvent.click(
```

Comment:
This is to ensure we can update the setting, if everything is not wired properly the test would fail, you can test that this this change works by reverting the change in `frontend/src/metabase/admin/settings/selectors.js` and this test should fail.

Reply:
Verified that the test will fail 👍🏻 

![CleanShot 2567-09-12 at 17 03 29@2x](https://github.com/user-attachments/assets/ee820eed-a281-4417-8530-3fc00cc7d98d)